gemviz23/demo/filterpanel.py

```diff
@@ -89,10 +89,15 @@ def filteredCatalog(self):
         if len(scan_id) > 0:
             cat = utils.get_tiled_runs(cat, scan_id=int(scan_id))
 
-        motors=self.positioner.text().strip()
+        motors=self.positioners.text().strip()
         if len(motors) > 0: 
-            for motor in motors.split(" "):
-                cat = cat.search(tiled.queries.Contains("motors", motor))
+            for motor in motors.split(","):
+                cat = cat.search(tiled.queries.Contains("motors", motor.strip()))
+
+        detectors=self.detectors.text().strip()
+        if len(detectors) > 0: 
+            for detector in detectors.split(","):
+                cat = cat.search(tiled.queries.Contains("detectors", detector.strip()))
```

Comment:
same comment here

Reply:
same thing

Reply:
Detector channel names at beam lines, despite our best insistence otherwise, sometimes contain whitespace.  So we need a delimiter that is not whitespace.  At least for detectors.  To be consistent, we use the same delimiter for both motors and detectors.

If we could use whitespace as a delimiter, the whole `split()` thing would be much easier:

```py
In [1]: "  m1   m2   ".split()
Out[1]: ['m1', 'm2']
```

So what happens if someone names their detector channel with this troublesome name `"I0, before #14"`?  After telling them not to do that, we'll have to re-think what to do.